src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/Jurisdiction.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+
+public enum Jurisdiction {
+    ENGLAND_AND_WALES("England and Wales"),
+    NORTHERN_IRELAND("Northern Ireland"),
+    SCOTLAND("Scotland"),
+    @JsonEnumDefaultValue
```

Comment:
If we agree with the JSON default value approach here, we'll have to go back and implement for the other fields that use enums, as they currently return a HTTP 500 error to the client when an invalid string value comes in.

Reply:
Created https://companieshouse.atlassian.net/browse/LP-492 to cover this for other fields.